### PR TITLE
add apiserver to tests

### PR DESCRIPTION
**Pull request by cseed**
Stacked on: https://github.com/hail-is/hail/pull/5384

Right now just running one test, since the Python tests are quite long (~10m).  When they get parallelized (ongoing batch changes), we enable the full tests.  The full tests are currently passing when run as:

    HAIL_TEST_SERVICE_BACKEND_URL=http://localhost:5000 gw testPython

### CODE REVIEW

**Comment by tpoterba** [comment]
On vacation until Friday, reassign if urgent!

**Comment by tpoterba** [comment]
Is this ready for review?

**Comment by cseed** [comment]
I have one thing left to do: I need to rebuild pr-builder because the apiserver test depends on Flask.  I'll do that shortly.  Rest of it is ready for review.

**Comment by cseed** [comment]
OK, should be ready to go.

File: hail/test-apiserver.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export PYSPARK_SUBMIT_ARGS="--conf spark.driver.extraClassPath=./build/libs/hail-all-spark.jar --conf spark.executor.extraClassPath=./build/libs/hail-all-spark.jar pyspark-shell"
+
+python ../apiserver/apiserver/apiserver.py >apiserver.log 2>&1 &
+server_pid=$!
+
+../until-with-fuel 30 curl -fL http://localhost:5000/healthcheck
+
+export HAIL_TEST_SERVICE_BACKEND_URL=http://localhost:5000
+
+python -m unittest test.hail.table.test_table.Tests.test_range_table
```
Comment (tpoterba) [inline hail/test-apiserver.sh]: I'm a bit confused about this line + the `@skip_unless_spark_backend` decorator. Do I correctly understand that this line is just necessary to keep testing times down at the moment (per your comment), but that we want the full test suite to run successfully against the service backend?

Comment (cseed) [reply hail/test-apiserver.sh]: Yes, exactly.  What's more, the full tests (minus the skipped ones) do in fact run successfully now.

Comment (tpoterba) [reply hail/test-apiserver.sh]: Okay, great.

**Pull request merged**